docs/src/content/docs/zh/guides/components.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: 组件
+description: 使用 Starlight 在 MDX 中使用组件
+---
+
+组件让你可以轻松地重用 UI 或样式。
+例如，链接卡片或 YouTube 嵌入。
+Starlight 支持在 [MDX](https://mdxjs.com/) 文件中使用组件，并提供了一些常用组件供你使用。
+
+[在 Astro 文档中了解更多关于构建组件的内容](https://docs.astro.build/zh-cn/core-concepts/astro-components/).
+
+## 使用组件
+
+你可以通过在 MDX 文件中导入组件，然后像渲染 JSX 标签一样来使用组件。
+这些看起来像 HTML 标签，但是以大写字母开头，与你的 `import` 语句中的名称匹配：
+
+```mdx
+---
+# src/content/docs/index.mdx
+title: 欢迎来到我的文档
+---
+
+import SomeComponent from '../../../components/SomeComponent.astro';
+import AnotherComponent from '../../../components/AnotherComponent.astro';
+
+<SomeComponent prop="something" />
+
+<AnotherComponent>
+  组件也可以包含**嵌套内容**。
+</AnotherComponent>
+```
+
+因为 Starlight 是由 Astro 提供支持的，所以你可以在 MDX 文件中添加对任何 [支持的 UI 框架（React、Preact、Svelte、Vue、Solid、Lit 和 Alpine）](https://docs.astro.build/zh-cn/core-concepts/framework-components/) 构建的组件的支持。
+在 Astro 文档中了解更多关于 [在 MDX 中使用组件](https://docs.astro.build/zh-cn/guides/markdown-content/#在-mdx-中使用组件) 的内容。
+
+### 与 Starlight 样式的兼容
+
+Starlight 为你的 Markdown 内容应用了默认样式，例如在元素之间添加边距。
+
+如果这些样式与你的组件的外观冲突，请在组件上设置 `not-content` 类来禁用它们。
+
+```astro
+---
+// src/components/Example.astro
+---
+
+<div class="not-content">
+	<p>不受 Starlight 默认内容样式的影响。</p>
+</div>
+```
+
+## 内置组件
+
+Starlight 为常见的文档用例提供了一些内置组件。
+这些组件可以从 `@astrojs/starlight/components` 包中获取。
+
+### 选项卡
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+你可以使用 `<Tabs>` 和 `<TabItem>` 组件显示一个选项卡界面。
+每个 `<TabItem>` 必须有一个 `label` 来显示给用户。
+
+```mdx
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+<Tabs>
+	<TabItem label="Stars">天狼星、织女星、参宿四</TabItem>
+	<TabItem label="Moons">木卫一、木卫二、木卫三</TabItem>
+</Tabs>
+```
+
+以上代码在页面上生成了以下选项卡：
+
+<Tabs>
+	<TabItem label="Stars">天狼星、织女星、参宿四</TabItem>
+	<TabItem label="Moons">木卫一、木卫二、木卫三</TabItem>
+</Tabs>
+
+### 卡片
+
+import { Card, CardGrid } from '@astrojs/starlight/components';
+
+你可以使用 `<Card>` 组件在与 Starlight 样式匹配的盒子中显示内容。
+当有足够的空间时，可以使用 `<CardGrid>` 组件将多个卡片封装在一起，以便并排显示卡片。
+
+`<Card>` 需要一个 `title`，并且可以选择包含一个 `icon` 属性，该属性设置为 [Starlight 内置图标](#所有图标) 之一的名称。
+
+```mdx
+import { Card, CardGrid } from '@astrojs/starlight/components';
+
+<Card title="看看这个">你想要突出显示的有趣内容。</Card>
+
+<CardGrid>
+	<Card title="Stars" icon="star">
+		天狼星、织女星、参宿四
+	</Card>
+	<Card title="Moons" icon="moon">
+		木卫一、木卫二、木卫三
+	</Card>
+</CardGrid>
+```
+
+以上代码在页面上生成了以下内容：
+
+<Card title="看看这个">你想要突出显示的有趣内容。</Card>
+
+<CardGrid>
+	<Card title="Stars" icon="star">
+		天狼星、织女星、参宿四
+	</Card>
+	<Card title="Moons" icon="moon">
+		木卫一、木卫二、木卫三
+	</Card>
+</CardGrid>
+
+:::tip[提示]
+在主页上使用卡片网格来显示项目的关键特性。
+添加 `stagger` 属性来垂直移动第二列的卡片，并增加视觉效果：
+
+```astro
+<CardGrid stagger>
+	<!-- cards -->
```

Comment:
This comment can probably be translated too. Maybe something like “Card components” would be better so you can still include a literal mention of “Card”?